docs/types/entity_action_types/drop_inventory.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Drop Inventory (Entity Action Type)
+date: 2022-6-07
+---
+
+# Drop Inventory
+
+[Entity Action Type](../entity_action_types.md)
+
+Drops items directly from any spot in a player's inventory.
+
+Type ID: `origins:drop_inventory`
+
+### Fields
+
+Field  | Type | Default | Description
+-------|------|---------|-------------
+`inventory_type` | [Identifier](../../data_types/identifier.md) | `inventory` | Takes one of either `inventory` or `power`. Determines whether you're dropping items from the player inventory, or the extra inventory from the [`origins:inventory`](../types/power_types/inventory.md) power type respectively.
+`entity_action` | [Entity Action](../types/entity_action_types.md) | _optional_ | If specified, this action will be ran after the items are dropped.
```

Comment:
```suggestion
`entity_action` | [Entity Action](../types/entity_action_types.md) | _optional_ | If specified, this action will be executed on the entity **before** the items are dropped.
```
The entity action type is actually executed *before* the item action type, which is also executed *before* dropping the item